packages/oidc-middleware/test/e2e/protractor.conf.js

```diff
@@ -12,10 +12,12 @@
 
 const JUnitXmlReporter = require('jasmine-reporters').JUnitXmlReporter;
 const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
+const { ProtractorBrowserLogReporter } = require('jasmine-protractor-browser-log-reporter');
 
 exports.config = {
   framework: 'jasmine2',
   onPrepare() {
+    jasmine.getEnv().addReporter(new ProtractorBrowserLogReporter());
```

Comment:
Where does the output of this show up?

Reply:
to the stdout. It will show in travis logs after all tests have run. https://github.com/thompsnm/jasmine-protractor-browser-log-reporter/blob/master/src/reporter.js#L30